server/daos/authorsBooks.js

```diff
@@ -1,45 +1,15 @@
 import db from '@database/models';
-import { isEmpty } from 'lodash';
 
-export const insertAuthorsBooks = async args => {
-  if (!isEmpty(args.authorsBooks)) {
-    const mapAuthorBooksArgs = args.authorsBooks.map((item, index) => ({
-      bookId: args.bookId,
-      authorId: item.authorId
-    }));
+export const insertAuthorsBooks = async args => db.authorsBooks.bulkCreate(args);
 
-    const res = await db.authorsBooks.bulkCreate(mapAuthorBooksArgs);
+export const updateAuthorsBooksForBooks = async args =>
+  db.authorsBooks.bulkCreate(args, {
+    fields: ['id', 'bookId', 'authorId'],
+    updateOnDuplicate: ['bookId']
```

Comment:
what would this do?

Reply:
this will check if the author has the bookId already present then it will skip that bookId

Reply:
makes sense, thank you